.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Great job adding the `test.md` to the gitignore file 👏🏻 Let's also add  the  `node_modules` directory  to  the gitignore file, ✍🏻